akita/__tests__/entity-store.spec.ts

```diff
@@ -62,17 +62,15 @@ describe('EntitiesStore', () => {
       store.add(new Todo({ id: 2 }));
       store.add(new Todo({ id: 3 }));
       store.add(new Todo({ id: 0 }), { prepend:true });
-      expect(store.entities[0].id).toEqual(0);
+      expect(Object.keys(store.entities)).toEqual([0, 1, 2, 3]);
     });
  
     it('should prepend with multi add', () => {
       store.add(new Todo({ id: 1 }));
       store.add(new Todo({ id: 2 }));
       store.add(new Todo({ id: 3 }));
       store.add([new Todo({ id: 0 }), new Todo({ id: 2 }), new Todo({ id: 44 })], {  prepend:true });
```

Comment:
you also added an entity with an id of 2.

Reply:
I don't understand? What can I do ? I should add more one entity with id of 2 as prepend?

Reply:
just use new ids.